### PR TITLE
Switch to the new account settings screen on OSS

### PR DESCRIPTION
**Pull request by bl-nero**
Contributes to https://github.com/gravitational/teleport/issues/36231

Screenshot 2024-01-10 at 19 14 20

Changelog: Implemented a new, redesigned account settings page.

### CODE REVIEW

File: web/packages/teleport/src/features.tsx [hunk 1]
```diff
@@ -58,8 +58,7 @@ const Audit = lazy(() => import('./Audit'));
 const Nodes = lazy(() => import('./Nodes'));
 const Sessions = lazy(() => import('./Sessions'));
 const UnifiedResources = lazy(() => import('./UnifiedResources'));
-const Account = lazy(() => import('./Account'));
-const AccountNew = lazy(() => import('./Account/AccountNew'));
+const Account = lazy(() => import('./Account/AccountNew'));
```
Comment (zmb3) [inline web/packages/teleport/src/features.tsx:61]: Will we eventually rename `AccountNew` to `Account` and get rid of the old one?

I don't like the idea of having `New` or `Next` on things. In a few years we'll end up with `AccountNewNew` etc.

Comment (bl-nero) [reply web/packages/teleport/src/features.tsx:61]: Yes, we will. I save it for the next week, though, as right now I have to prioritize squeezing in more impactful changes.

Comment (kimlisa) [reply web/packages/teleport/src/features.tsx:61]: `AccessRequestRequest`

**Review by kimlisa — APPROVED** [review]

**Comment by bl-nero** [comment]
> This would be a good one to check against the test plan to see if any updates/additions are needed.

@zmb3 I looked at the test plan, and it's generic enough that nothing needs to be updated.

**Review by zmb3 — APPROVED** [review]

**Comment by bl-nero** [comment]
(Giving it another try, since the test in question looks unrelated. Might be just a flake.)

**Pull request merged**